functionary/ui/templates/core/package_list.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'base.html' %}
+
+{% block menu_select %}{% url 'ui:package-list' %}{% endblock menu_select %}
+{% block content %}
+
+<div>
+    {% for package in object_list %}
+    <div class="block">
+        <div class="card">
+            <div class="card-content">
+                <div class="media">
+                    <div class="media-left">
+                        <span class="icon is-large">
+                            <i class="fa fa-2x fa-box-open"></i>
+                        </span>
+                    </div>
+                    <div class="media-content">
+                        <a href="{% url 'ui:package-detail' package.id %}">
+                            <p class="title is-4">
+                                {% if package.display_name%}
```

Comment:
Just because it'll drive me nuts, I'm going to make a note here that we should at some point look into formatters that are compatible with django templating.  I don't know that there's a super obvious solution out there, but it'd save us from little inconsistencies, like the lack of a space before the closing `%}` here.

Reply:
Added djlint to the project. Use the following to run it:
`djlint functionary/ui/templates --profile=django`